### ISSUE TITLE
Problem to require in Laravel 11

### ISSUE THREAD

**Issue by meWasim** [lazychaser/laravel-nestedset#598]
Problem 1
    - kalnoy/nestedset 1.0 requires illuminate/support >=4.0,<4.2 -> found illuminate/support[v4.0.0, ..., v4.1.30] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v1.1 requires illuminate/support ~4.2.1 -> found illuminate/support[v4.2.1, ..., v4.2.17] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v2.0, ..., v2.1.0] require illuminate/support ~4.1 -> found illuminate/support[v4.1.0, ..., v4.2.17] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v2.2.0, ..., v2.4.2] require illuminate/support >=4.1,<5.1.0 -> found illuminate/support[v4.1.0, ..., v4.2.17, v5.0.0, ..., v5.0.33] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v2.4.3, ..., v2.4.4, v3.0.0] require illuminate/support >=4.1,<5.2.0 -> found illuminate/support[v4.1.0, ..., v4.2.17, v5.0.0, ..., v5.1.41] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v3.1.0, ..., v3.1.4] require illuminate/support ~5.1.0 -> found illuminate/support[v5.1.1, ..., v5.1.41] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v4.0.0, ..., v4.1.4] require illuminate/support 5.2.* -> found illuminate/support[v5.2.0, ..., v5.2.45] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[4.1.5, ..., v4.2.0] require illuminate/support 5.2.*|5.3.x -> found illuminate/support[v5.2.0, ..., v5.3.23] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v4.2.1, ..., v4.2.7] require illuminate/support 5.2.* | 5.3.* | 5.4.* -> found illuminate/support[v5.2.0, ..., v5.4.36] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v4.3.0, ..., v4.3.1] require illuminate/support 5.2 - 5.5 -> found illuminate/support[v5.2.0, ..., v5.5.44] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v4.3.2 requires illuminate/support 5.2 - 5.6 -> found illuminate/support[v5.2.0, ..., v5.6.39] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v4.3.3 requires illuminate/support 5.2 - 5.7 -> found illuminate/support[v5.2.0, ..., v5.7.28] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v4.3.4 requires illuminate/support 5.2 - 5.8 -> found illuminate/support[v5.2.0, ..., v5.8.36] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v4.3.5 requires illuminate/support ~5.7.0|~5.8.0 -> found illuminate/support[v5.7.0, ..., v5.8.36] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v5.0.0 requires illuminate/support ~5.7.0|~5.8.0|~6.0 -> found illuminate/support[v5.7.0, ..., v5.8.36, v6.0.0, ..., v6.20.44] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v5.0.1 requires illuminate/support ~5.7.0|~5.8.0|^6.0|^7.0 -> found illuminate/support[v5.7.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v5.0.2, ..., v5.0.3] require illuminate/support ~5.7.0|~5.8.0|^6.0|^7.0|^8.0 -> found illuminate/support[v5.7.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v5.0.4 requires illuminate/support ~5.7.0|~5.8.0|^6.0|^7.0|^8.41 -> found illuminate/support[v5.7.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.41.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v5.0.5 requires illuminate/support ~5.7.0|~5.8.0|^6.0|^7.0|>=8.0 <8.43.0 -> found illuminate/support[v5.7.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.42.1] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v6.0.0 requires illuminate/support ^7.0|^8.0 -> found illuminate/support[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset v6.0.1 requires illuminate/support ^7.0|^8.0|^9.0 -> found illuminate/support[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16] but these were not loaded, likely because it conflicts with another require.
    - kalnoy/nestedset[v6.0.2, ..., v6.0.3] require illuminate/support ^7.0|^8.0|^9.0|^10.0 -> found illuminate/support[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.3] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires kalnoy/nestedset * -> satisfiable by kalnoy/nestedset[1.0, v1.1, v2.0, ..., v2.4.4, v3.0.0, ..., v3.1.4, v4.0.0, ..., v4.3.5, v5.0.0, ..., v5.0.5, v6.0.0, v6.0.1, v6.0.2, v6.0.3].

**Comment by fomvasss** [comment]
Yes, I confirm, the problem of dependencies when installing on Laravel 11 (

**Comment by TraLeeee** [comment]
~~It's currently available when you install from dev channel.~~
~~`composer require kalnoy/nestedset:v6.x-dev`~~
**Update 8/4/2024:**
The author already bump to v6.0.4 :rocket: No need to use dev channel anymore!
Just run: `composer require kalnoy/nestedset`

**Comment by dmitrijsmihailovs** [comment]
@lazychaser Please, release a new tag for the latest merged PR. Thank you 👍

**Comment by glennjacobs** [comment]
@lazychaser any update on tagging a new release? Thanks.

**Comment by Rubelsoldier** [comment]
Yesterday, I tried to download via this url, composer require kalnoy/nestedset, but got the above errors. Any solutions came?

**Comment by shadykip** [comment]
@lazychaser  I am facing the same issue with laravel 12
